.github/workflows/cron-job-its.yml

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: Cron Job ITs
```

Comment:
whats the reason for these changes?

Reply:
Just to make naming consistent across workflows.